### ISSUE TITLE
Browser extension hovers are broken on GitHub PRs (for Refined GitHub users)

### ISSUE THREAD

**Issue by sqs** [sourcegraph/sourcegraph#2957]
It looks like GitHub's PR DOM changed for users of Refined GitHub. See the screenshots in the next comment for the A/B test.

Repro:

1. Go to a PR, such as https://github.com/gorilla/mux/pull/463/files or https://github.com/sourcegraph/go-diff/pull/21/files, and hover over tokens

Expected: hovers

Actual: no hovers, and error in JS console:

```
inject.bundle.js:1 Uncaught TypeError: Cannot read property 'getElementsByTagName' of undefined
    at p (inject.bundle.js:1)
    at getDiffCodePart (inject.bundle.js:1)
    at inject.bundle.js:1
    at t.project (inject.bundle.js:1)
    at t._next (inject.bundle.js:1)
    at t.next (inject.bundle.js:1)
    at t.notifyNext (inject.bundle.js:1)
    at t._next (inject.bundle.js:1)
    at t.next (inject.bundle.js:1)
    at t._next (inject.bundle.js:1)
```

**Comment by sqs** [comment]
It seems to be working in another Chrome browser profile. Will try to narrow down the problem.

**Comment by sqs** [comment]
...and now it's back to the old split diff toggle UI for me on my main GitHub account, so I am not experiencing this problem anymore.

@lguychard @felixfbecker Feel free to close this if you don't find it useful. I suspect this means that GitHub's DOM will soon change here, so it might be good to add robustness in advance.

**Comment by felixfbecker** [comment]
@sqs just to be sure, it's not Refined GitHub?

**Comment by sqs** [comment]
@felixfbecker Oh yeah, you are right, it is. I updated the title to reflect that. We have a fair number of users of Refined GitHub, so it is still important to fix (but less important than #2958).

**Comment by felixfbecker** [comment]
Agreed
